### DISCUSSION TITLE
How do I use tilemaps from replit?

### DISCUSSION THREAD

**Discussion by bestscarper** [replit/kaboom#181] [Q&A] [answered]
In order to learn about using the replit kaboom environment I tried to port the "platformer" example, but immediately hit a problem.

If the tilemap is added to the "sprites" folder, the "loadSprite" code is generated but it doesn't allow for the extended spriteconf required to slice the timemap.  If I try to edit the generated html file it gets replaced.

Even if I try to load the spritemap directly from the kaboom.js domain I just get "sprite not found" triggered on the invocation of ```sprite("tilemap",...)```.

```
loadSprite("tilemap","https://kaboomjs.com/pub/examples/tilemaps/bit-platformer.png",{
	sliceX: 20,
	sliceY: 20,
	anims: {
		idle: {from: 300, to: 300},
		run: {from: 301, to: 305},
	},
});

// define player object
const player = add([
	sprite("tilemap", {
		animSpeed: 0.05,
		frame: 300,
	}),
	area(vec2(-4, 0), vec2(4, 8)),
	pos(TILE_W * 6, 0),
	color(0.5, 0.5, 1),

	origin("center"),
	scale(SCALE),
	body(),
]);
```

**Comment by slmjkdbtl** [comment, marked as answer]
Yep! A brand new replit environment will be released next week which doesn't generate these code along with a lot of other improvements. For now if want to use the tilemap you might have to use the empty html template
